sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -545,6 +545,16 @@ class SQLQuerySuite extends QueryTest with SharedSQLContext {
     }
   }
 
+  test("SPARK-28002: CTE with column alias") {
+    checkAnswer(
+      sql("""
+        |WITH t(x) AS (SELECT 1)
+        |SELECT * FROM t WHERE x = 1
+        """.stripMargin),
+      Row(1) :: Nil
+    )
+  }
```

Comment:
Oh, please move this to `cite.sql`. That is a perfect place for this.
cc @gatorsmile

Reply:
ok, moved